runtime/doc/options.txt

```diff
@@ -3663,9 +3663,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Example: >
 	    :set fillchars=stl:\ ,stlnc:\ ,vert:\|,fold:-,diff:-
 <
-	For the "stl", "stlnc", "foldopen", "foldclose" and "foldsep" items
-	single-byte and multibyte characters are supported.  But double-width
-	characters are not supported. |E1512|
+	All items single-byte and multibyte characters are supported.  But
```

Comment:
This is not clear: "All items"

Reply:
"All items" are `All items`.
![image](https://github.com/user-attachments/assets/9e4aff01-fde1-40e7-ad2f-256f0eec0e63)

Reply:
```suggestion
	All items support single-byte and multibyte characters.  But
```

Reply:
That's better.